README.md

```diff
@@ -131,13 +131,16 @@ require'nvim-treesitter.configs'.setup {
     disable = { "c", "rust" },
     -- Or use a function for more flexibility, e.g. to disable slow treesitter highlight for large files
     disable = function(lang, buf)
-        local max_filesize = 100 * 1024 -- 100 KB
-        local ok, stats = pcall(vim.loop.fs_stat, vim.api.nvim_buf_get_name(buf))
-        if ok and stats and stats.size > max_filesize then
-            return true
-        end
+      local max_filesize = 100 * 1024 -- 100 KB
+      local filename = vim.api.nvim_buf_get_name(buf)
+      local file_size = vim.fn.getfsize(filename)
+
+      if file_size > max_filesize then
+        return true
+      end
```

Comment:
`getfsize` returns -1, which doesn't seem to be handled here?

Additionally, why not just move vim.loop to vim.uv now that vim.uv is in latest release version of Neovim?